core/test/unit/lib/mobiledoc/cards/markdown_spec.js

```diff
@@ -4,51 +4,14 @@ const SimpleDom = require('simple-dom');
 const serializer = new SimpleDom.HTMLSerializer(SimpleDom.voidMap);
 
 describe('Markdown card', function () {
-    describe('version 1', function () {
```

Comment:
Are we OK removing the v1 tests if we're still allowing rollback using the v1 renderers?

Reply:
Yeah i think it's fine to remove them. We only need the v1 renderer for a db rollback.
Do you agree? Let me know :)

Reply:
Or we keep them and mark them as deprecated 🤔 Hm maybe better. I always do it like this.

Reply:
I'll keep one. Will push a commit.

Reply:
https://github.com/TryGhost/Ghost/pull/9750/commits/cba3e0a936bb5ce7281ff8e6d4b295e0af9b969d